membrane/client/flask/__init__.py

```diff
@@ -0,0 +1,244 @@
+import json
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from functools import wraps
+from urllib.parse import urljoin
+
+import jwt
+from flask import Blueprint, Flask, redirect, request, url_for
+from flask_login import LoginManager, UserMixin
+from flask_login import current_user as membrane_current_user
+from flask_login import login_required as _login_required
+from flask_login import login_user, logout_user
+
+ALGORITHM = "RS256"
+DEFAULT_TOKEN_EXPIRATION = 300
+DEFAULT_LANDING_URL = "/"
+DEFAULT_LOGGED_OUT_URL = "/"
```

Comment:
I think I need an explanation about how this work between landing page and logged out page. I'm suspicious that these are the same value by default and how that works so I need a demo.